onnxruntime/core/graph/contrib_ops/onnx_function_util.cc

```diff
@@ -102,8 +102,61 @@ bool BuildFunctionProto(FunctionProto& functionProto, const OpSchema& schema,
                         const std::vector<FunctionBodyHelper::NodeDef>& node_defs,
                         const std::vector<OperatorSetIdProto>& relied_opsets) {
   BuildNodes(functionProto, node_defs);
-  schema.BuildFunction(functionProto, relied_opsets);
+  for (auto& relied_opset : relied_opsets) {
```

Comment:
Now that this function has been moved to onnx, can't we delete this here (and call the onnx version)?

Reply:
Delete previous function too? And call the one in FunctionBodyHelper?